pkg/scaling/scale_handler.go

```diff
@@ -237,7 +237,6 @@ func (h *scaleHandler) startPushScalers(ctx context.Context, withTriggers *kedav
 		go func(s scalers.PushScaler) {
 			activeCh := make(chan bool)
 			go s.Run(ctx, activeCh)
-			defer s.Close(ctx)
```

Comment:
The close method has already been called once in ScalersCache. PushScaler here will call the scaler's close method again.

Reply:
Could you please open a separate PR for this change?

Reply:
> Could you please open a separate PR for this change?

done!

Reply:
PR->https://github.com/kedacore/keda/pull/3599#issue-1351194353